how-to/web-interop-support-context-and-intents/README.md

```diff
@@ -23,7 +23,7 @@ The platform also supports:
 
 ## Layout
 
-To show so much content we implemented a layout override so that we can:
+To show so much content we implemented a layoutManagerOverride so that we can:
```

Comment:
One more nit: `To show so much content` - why the "so much" wording for content?
Should it be:
`To show content in a multi-layout UI we implemented a layoutManagerOverride so that we can:`

Reply:
Applied.